### PR TITLE
*.eclass: Move EXPORT_FUNCTIONS below inherit 

### PR DESCRIPTION
**Pull request by ulm**

### CODE REVIEW

**Comment by ulm** [comment]
@gentoo/apache
@gentoo/base-system 
@gentoo/gnustep 
@hattya
@gentoo/java
@junghans
@gentoo/kde
@gentoo/kernel
@gentoo/php
@gentoo/sci-biology

**Review by mgorny — APPROVED** [review]
LGTM

**Comment by a17r** [comment]
Please drop ecm.eclass and kde.org.eclass, both are already covered by pending EAPI-8 support PR.

File: eclass/gnustep-base.eclass [hunk 1]
```diff
@@ -263,3 +261,6 @@ EOF
 }
 
 fi
+
+EXPORT_FUNCTIONS pkg_setup src_prepare src_configure src_compile src_install \
+	pkg_postinst
```
Comment (SoapGentoo) [inline eclass/gnustep-base.eclass:266]: would have preferred a single line, makes for easier grepping

**Review by SoapGentoo — APPROVED** [review]

**Comment by ulm** [comment]
Merged.

**Pull request closed without merging**